app/src/main/java/com/github/epfl/meili/poi/PoiActivity.kt

```diff
@@ -8,12 +8,11 @@ import androidx.viewpager2.widget.ViewPager2
 import com.github.epfl.meili.R
 import com.github.epfl.meili.map.MapActivity
 import com.github.epfl.meili.models.PointOfInterest
+import com.github.epfl.meili.util.MenuActivity
 
-class PoiActivity : AppCompatActivity() {
+class PoiActivity : MenuActivity(R.menu.nav_poi_info_menu) {
```

Comment:
Now that we are not using the viewpager anymore, we can combine this and PoiInfoFragment into a single activity I think